docs/content/guides/getting-started/demo.md

```diff
@@ -12,21 +12,36 @@ tags:
 
 [[toc]]
 
+::: only-for javascript
 Play around with a demo of Handsontable, in your favorite framework.
+:::
+
+::: only-for react
+Play around with a demo of Handsontable, in React.
```

Comment:
Correct me if I'm wrong, but that's not exactly true, right? The demo displayed below is a vanilla version?